omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitController1_9Test.java

```diff
@@ -214,4 +214,32 @@ public void searchByPatient_shouldGetUnretiredVisitsForThePatient() throws Excep
 		Assert.assertEquals(3, Util.getResultsSize(result));
 	}
 	
+	@Test
+	public void searchByPatient_shouldGetRetiredVisitsIfIncludeAllIsTrue() throws Exception {
+		String patientUUid = "da7f524f-27ce-4bb2-86d6-6d1d05312bd5";
+		int visitIncludingVoidedSize = 0;
+		int visitWithoutVoidedSize = 0;
```

Comment:
If you named one as without, then the other would have with.
If you choose including, then the other would have excluding.
That way it is easier to see the opposite of the other. Does this make sense?

Reply:
Yes